ONSdigital/design-system
# Remove padding on warning panel

### What is the context of this PR?
Fixes: #1194 

This is a **BREAKING CHANGE** for UAC. The utility classes `u-mb-l` and `u-mt-l` will now need to be passed into the panel in the collapsible for that pattern like the change in this PR for the UAC example.

### How to review 
- Check padding has been removed
- Check everything that uses the panel still looks correct
